presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -225,6 +228,20 @@ public SystemSessionProperties(
                         COLOCATED_JOIN,
                         "Experimental: Use a colocated join when possible",
                         featuresConfig.isColocatedJoinsEnabled(),
+                        false),
+                new PropertyMetadata<>(
+                        OPERATOR_MEMORY_LIMIT_BEFORE_SPILL,
+                        "Experimental: Operator memory limit before spill",
+                        VARCHAR,
+                        DataSize.class,
+                        featuresConfig.getOperatorMemoryLimitBeforeSpill(),
+                        false,
+                        value -> DataSize.valueOf((String) value),
+                        DataSize::toString),
+                longSessionProperty(
+                        MAX_ENTRIES_BEFORE_SPILL,
+                        "Experimental: Limit max number of entries before spill",
```

Comment:
I thought we decided to remove this property, and just have a property to turn on spills and size the buffer used when spilling?